x/margin/client/cli/tx_open.go

```diff
@@ -63,6 +85,8 @@ func CmdOpen() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().String("take-profit", types.InfinitePriceString, "Optional take profit price")
```

Comment:
Can you define the flag variable to somewhere and use it rather than pure string in two places?

Reply:
done